views/features.erb

```diff
@@ -23,12 +23,16 @@
 				<li>It’s resilient since it operates across three independent availability zones</li>
 				<li>It’s built on an open source PaaS platform (Cloud Foundry)</li>
 				<li>You can decide to move to other Cloud Foundry-based PaaS providers, including on GCloud, as you’re not locked into our platform</li>
+				<li>It's best suited for <a href="https://12factor.net/">Twelve-Factor App development</a></li>
```

Comment:
I don't like this wording. I hope that's helpful.

Reply:
![](https://media.giphy.com/media/j1oCUFCeCdClq/giphy.gif)